<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddPerson</class>
 <widget class="QDialog" name="DialogAddPerson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodaj Osobę</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Imię</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="nameInput"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="surnameLabel">
        <property name="text">
         <string>Nazwisko</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="surnameInput"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="deadLabel">
        <property name="text">
         <string>Żyje</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QCheckBox" name="aliveCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="bornLabel">
        <property name="text">
         <string>Urodzony</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="bornDateInput">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="deathDateInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="placeLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>103</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>103</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="Polish" country="Poland"/>
        </property>
        <property name="text">
         <string>Miejsce zamieszkania</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QLineEdit" name="placeInput"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="deathLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Zmarły</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameInput</tabstop>
  <tabstop>surnameInput</tabstop>
  <tabstop>aliveCheckBox</tabstop>
  <tabstop>bornDateInput</tabstop>
  <tabstop>deathDateInput</tabstop>
  <tabstop>placeInput</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogAddPerson</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogAddPerson</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
